crates/engine/src/prepare/operation/before_variables.rs

```diff
@@ -13,15 +13,22 @@ impl<'ctx, R: Runtime> PrepareContext<'ctx, R> {
         request: &Request,
         document: &str,
     ) -> PrepareResult<CachedOperation> {
-        let parsed_operation = crate::operation::parse(self.schema(), request.operation_name.as_deref(), document)?;
+        // TODO: Make the number here configurable probably?
```

Comment:
ah nice was thinking of this. we have `request_body_limit` in the config which serves a similar purpose, just it's for the whole body.